From 5c8be1e343e93672ad23a65a2f32b866f625e36d Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 29 Oct 2026 09:06:53 +0100
Subject: [PATCH 1/2] Update the upstream signing key

Update B5B6416FEA6DDA05EA562A9FCB987F2783972FF9 to the current copy at
https://keys.openpgp.org/vks/v1/by-fingerprint/B5B6416FEA6DDA05EA562A9FCB987F2783972FF9,
the URL the spec names for it (same certificate(s)).

The current key is needed to verify the release signature with
openpgpverify (sqv), which checks that the key was valid when the
release was signed.

Assisted-by: Claude Opus 5.5
---
 B5B6416FEA6DDA05EA562A9FCB987F2783972FF9 | 51 ++++++++++++++++++------
 1 file changed, 38 insertions(+), 13 deletions(-)

diff --git a/B5B6416FEA6DDA05EA562A9FCB987F2783972FF9 b/B5B6416FEA6DDA05EA562A9FCB987F2783972FF9
index 06ab48a..16c8846 100644
--- a/B5B6416FEA6DDA05EA562A9FCB987F2783972FF9
+++ b/B5B6416FEA6DDA05EA562A9FCB987F2783972FF9
@@ -15,17 +15,42 @@ VV2RvjjDBhbyuMllYfzPqxs9w7pSrnt6z4BoHMAOWVut6yzQM8khLLcLKY/2pFVo
 qB8hqJ1usdkhlHdbqYKYlFOS6TwpT/61IXG/uqFzUKRcloq2MBBGH6Xk6QARAQAB
 zTxTZWJhc3RpYW4gQmVub2l0IChzb2Z0d2FyZSBzaWduaW5nIG9ubHkpIDxiZW5u
 b0BvcGVuYnNkLm9yZz7CwZQEEwEIAD4CGwMFCwkIBwIGFQoJCAsCBBYCAwECHgEC
-F4AWIQS1tkFv6m3aBepWKp/LmH8ng5cv+QUCZBdXugUJCyYSiQAKCRDLmH8ng5cv
-+W4NEACOyrljdrlCRu4oUaIuxCp8dX8xbVhv72sjptrz5vfnsKO65saR8r3iyQLX
-K3R0Vlx4DseXrr7vJBBicBNdWl0i02Cxz3r4EugJ+SLfkzJesUQR9MJQIB7PjSWB
-XFLw4VH/BeuUMS1yZXdfvpTK5riGrFaNW+lLSi/ZTuNR8I3OWeG8SeLWhVFPbMlV
-evHfEWZpD5cGc7heqV3HCuVYJd9oWxI9T+To+laFA6PQ65s/sxZydSdToHvh/r0S
-5NMNBWBK29InNkuGhoe1K6NDwhJMN0a3YgkjsvopZZos8gDO0Enu++C0KTdivJYa
-g9p7xT65rKqsPc/y9V97gt7IEgA+4rpfT1PpOYVl3+fOEwdK8ynW5hjlH0/7xyqq
-jrzGsfbL0fO9MXyoXbbBlc61rz5J+6miQoykfiZyctqWQ7xsoK9CjYxtGb4wbuIP
-oSYLY962Lo7lwIRM7pu9ct/qoTlAm9EEzgZKS7I+K+Fkqa+qJwKCjflf6tjN42vZ
-Eya1d7YJzm1COGffMvS/J8poYHwvGN1mRO2R6iPIpgWJvKNa/yQeLj1RZwKGmCSe
-1jL3FL/kx3IVJc9tFUaSTZ/yYnYq5WTVyXAlDL++fBSGVB6WMk+t0HK5TBDaOAp4
-CsWpomwxECzyBwmJ9UP3nE0suls74KSlsrIsURVujkH6Keb7Dg==
-=4Gk8
+F4AWIQS1tkFv6m3aBepWKp/LmH8ng5cv+QUCad6cEwUJDwwjYgAKCRDLmH8ng5cv
++S0ZD/sFwTJTZ1Kc/5wq+ypN0CgRGqcGyFC9bRKdGlrflfsSrzgT3qAJVAIyMG9r
+n6DGT1jelw/Ntt8VxoxAhDpMYgqV00ciBwZ8iucdeGHAf1dH+miqbhB7mJ4YzgZ9
+AGIiNRq5RLyEn1+ae7z0zOmaKUWRCBwkeRiMkBCF+Ozj+I5qO3saqgPIwWmbrpLu
+YCg8JZp27tqFpzwO0kXDUEopTIcaDm6Jzv3ZFdVTTlH+vsZa4G4+aBYoD60Tiros
+CslXLdTCz2Tso3NJI4Qrd7tvvtA2OPIv/UwAKu8SP2v9f7ghJoRGeacN8ttyleZt
+pwSFVXEypSrFWhp2jd8df+Xk4QIKPrym2cHCTJ1QDuzAkIE8tb6lu6Eg4lY4VRx8
+o3HvpfE7kcQh30LU/8IAxT7p5/jJCa0n5qzKam49/cNthwoHuuKKM8JSiWefXU5o
+sSNCJBUSJAT6yDfKxIYPVKpFvggj8xg23dL+0X4N9u936jy3ebSyCdzh4JJmyAu+
+FRux8BB4oFPDMK3X5S2KdT4aBHeQfkvjLhbqZNGSGdKV0sJralyJ7n5ttRMlv6eT
+/6At94wSiCyVyhAbmGUmmG3Ao96XVi1CD2K7wqtc87yepnkNB/VE3b//CLaDL4xQ
+LX2AWNVihQnDw3SEpp/O+doey68f6OZ4O4XCV4DdKF07CgFR0MLBlAQTAQgAPgIb
+AwULCQgHAgYVCgkICwIEFgIDAQIeAQIXgBYhBLW2QW/qbdoF6lYqn8uYfyeDly/5
+BQJkF1e6BQkLJhKJAAoJEMuYfyeDly/5bg0QAI7KuWN2uUJG7ihRoi7EKnx1fzFt
+WG/vayOm2vPm9+ewo7rmxpHyveLJAtcrdHRWXHgOx5euvu8kEGJwE11aXSLTYLHP
+evgS6An5It+TMl6xRBH0wlAgHs+NJYFcUvDhUf8F65QxLXJld1++lMrmuIasVo1b
+6UtKL9lO41Hwjc5Z4bxJ4taFUU9syVV68d8RZmkPlwZzuF6pXccK5Vgl32hbEj1P
+5Oj6VoUDo9Drmz+zFnJ1J1Oge+H+vRLk0w0FYErb0ic2S4aGh7Uro0PCEkw3Rrdi
+CSOy+illmizyAM7QSe774LQpN2K8lhqD2nvFPrmsqqw9z/L1X3uC3sgSAD7iul9P
+U+k5hWXf584TB0rzKdbmGOUfT/vHKqqOvMax9svR870xfKhdtsGVzrWvPkn7qaJC
+jKR+JnJy2pZDvGygr0KNjG0ZvjBu4g+hJgtj3rYujuXAhEzum71y3+qhOUCb0QTO
+BkpLsj4r4WSpr6onAoKN+V/q2M3ja9kTJrV3tgnObUI4Z98y9L8nymhgfC8Y3WZE
+7ZHqI8imBYm8o1r/JB4uPVFnAoaYJJ7WMvcUv+THchUlz20VRpJNn/JidirlZNXJ
+cCUMv758FIZUHpYyT63QcrlMENo4CngKxamibDEQLPIHCYn1Q/ecTSy6WzvgpKWy
+sixRFW6OQfop5vsOwsGUBBMBCAA+AhsDBQsJCAcCBhUKCQgLAgQWAgMBAh4BAheA
+FiEEtbZBb+pt2gXqViqfy5h/J4OXL/kFAmB4mAoFCQWmH1kACgkQy5h/J4OXL/mp
+Uw/+PLJZzCedk2tuMo6kaItngZ49jgjtFiXJ+8dslx5kG6HlD7m73hb+R71g4XnV
+bwMH/Y5PckJxKCpMu9SjZYd8eYpVvlzOc0K2R84CwIAK1R81J8eMDAIvdk7ggPoj
+EZy94vbBNvANp1BrugpOpkMJDnQ9YQ/m2vFAPKkFQKFfgIIOwjZJrgHz39j/K6vB
+6PS5+GVsBX6roCefNN6IiumYByG/PB8e0ZJSw1xuiRtgSDHksEg8t1qXNyhHxajr
+jUrWrxm4RK4tD1hInxk2HB3NHKkXHPd05/dcJUq3FUW0QiDysNN5MSUD2VdkCXpI
+MiRioXBIxS4gxWP4m8jaUnTagT8pwTgX3GNdZfle3R+PT2isd+2mMKy00B3L0HrI
+aY8VFubgjIP+LPqnf0K7XQLjKf19zKhits0ulo1H7xLyka2wDn3/Iiyb+v4nrrx1
+QB4Wg+NREmBJ90uaWq1BWrOx1uLUUEF30Lab9xyMNTt6y8eJX+wan10bQegZSUBo
+HqKb/fng7VwJAHXWcowAKzameZCZWbQt//R/FCKmABko5/jSuOrgeD/iNiP0vhsy
+NUt8gbMPtRAPBY2EE8os9xUqACEoJC6C9Q//a+M0+WR+Z0ACBqVyIVL8nSUjWxys
+LR4ioOm6nkKRjyKXHvp/btZ7hB05yLfPLNKpuj//5JIUdkY=
+=+XCU
 -----END PGP PUBLIC KEY BLOCK-----
-- 
2.56.0


